src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/BasicCoercion.txt

```diff
@@ -35,10 +35,10 @@ true
 
 // can coorce text to bool via logical operators
 >> "true" && "anything"
-false
+#Error
```

Comment:
(Kind=InvalidArgument) - and can you also update the function InvalidBooleanFormatError in CommonErrors.cs to use this kind? Currently it uses 'BadLanguageCode', which is incorrect.

Reply:
done